app/main.py

```diff
@@ -14,4 +14,10 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+
+    counter = 0
+    for character in phrase:
+        if character.lower() == letter.lower():
+            counter += 1
+
+    return counter
```

Comment:
1. The solution is ok, but can you try to rewrite it using just 1 line?
2. Always add a blank line at the end of the file.